src/hooks/useKeyboardNavigation.js

```diff
@@ -27,11 +29,68 @@ const useKeyboardNavigation = ({
 
   useEffect(() => {
     return tinykeys(emojiListRef.current, {
-      ArrowRight: focusNextEmoji,
-      ArrowLeft: focusPrevEmoji,
+      ArrowRight: () => navigateGrid(RIGHT),
+      ArrowLeft: () => navigateGrid(LEFT),
+      ArrowUp: () => navigateGrid(UP),
+      ArrowDown: () => navigateGrid(DOWN),
```

Comment:
These functions should be wrapped with try/catch. Same for the functions in the other tinyKeys implementations.

1. What happens when for some reason our assumptions are wrong?
For example - in `const baseOffset = gridChildren[0].offsetTop;` what if gridChildren is empty?
2. There are cases when navigating with the keyboard doesn't make sense. For example - hitting "DOWN" in the search box when there are no results (hint - we'll get an error)
![image](https://user-images.githubusercontent.com/11255103/151076370-1ba92ae4-07c6-4d6f-941c-3490465ffdaa.png)

Reply:
Hi ,  where did you want the try catch to be, inside the navigateGrid, or when we call it?
should it be something like this?
` ArrowUp: () => {
        try {
          navigateGrid(UP);
        } catch (error) {
          console.error(error);
        }
      },`

Reply:
a. no need to console
b. you can do something like `withCatch`

function withCatch(cb) {
  return (...args) => 
	try {
		cb(...args)
	}  catch {}
}